CMakeLists.txt

```diff
@@ -23,9 +23,10 @@
 #
 #=============================================================================
 
-cmake_minimum_required(VERSION 2.8.12 FATAL_ERROR)
+cmake_minimum_required(VERSION 3.15.2 FATAL_ERROR)
```

Comment:
Why bump this up? Which features requires a newer cmake?

Reply:
No deep reason. I just have not tested on other versions. Feel free to downgrade to the earliest version you know it builds properly.

Reply:
I think the opposite way: I think it's better to keep it as is until we know there is a feature we need from higher versions.